examples/nlp/text_classification_with_switch_transformer.py

```diff
@@ -78,8 +78,8 @@ def __init__(self, maxlen, vocab_size, embed_dim):
         self.pos_emb = layers.Embedding(input_dim=maxlen, output_dim=embed_dim)
 
     def call(self, x):
-        maxlen = ops.shape(x)[-1]
```

Comment:
We should use `ops` rather than `tf`. I don't understand what this change accomplishes. The example already runs on Keras 3.

Reply:
Okay then let's close this pr.